### PR TITLE
popular developer searcher added

### LINKED ISSUE #30: Add search by developer (popular/recently joined)
Using [js-search](https://github.com/bvaughn/js-search) we want to search by:
* login 
* name

The result should be sorted **followers** desc.

Note: We should fetch the data from [users.json](https://developersdo.github.io/opensource/data/users.json)
This issue depends on #8

### PR DESCRIPTION
**Pull request by ljcordero**
- [x] Popular

- [ ] Recently Joined

Closes #30

### CODE REVIEW

**Comment by ljcordero** [comment]
@DannyFeliz may you please confirm me please if the 'forked' property on users.json is working and also if it is right to use in the Developers view.

**Comment by DannyFeliz** [comment]
> @DannyFeliz may you please confirm me please if the 'forked' property on users.json is working and also if it is right to use in the Developers view.

I will change the data source soon with all the necessary information. I was talking with @rmariuzzo and he created a new [repository](https://github.com/developersdo/opensource-data) for the data and recently he added the repositories data, so, let's leave it like this for now until we fix the that.

**Comment by ljcordero** [comment]
The last thing,

The grid system broke with some specific width: e.g.

image

I have tried many things, and the only one which works for me was a fixed height, but don't like use it, if you can help me I will thanks you.

@DannyFeliz @larizzatg

**Comment by DannyFeliz** [comment]
Let's wait for @larizzatg to take a look

**Comment by ljcordero** [comment]
Ok, I have fixed the columns display in this commit 6fff5b0b0b5ed56c2442be03f3a4564008e5aa51

And we are ready to code review. 👯

File: src/views/PopularDevelopers.vue [hunk 1]
```diff
@@ -1,12 +1,68 @@
 <template>
   <div>
     <h3 class="center-align">Popular Developers</h3>
-    <slot />
+    <p class="center">
+      Showing
+      <strong>{{ filteredDevelopers.length.toLocaleString() }}</strong> developers
```
Comment (DannyFeliz) [inline src/views/PopularDevelopers.vue:6]: Actually, here we show the total of developers here.
```suggestion
      <strong>{{ developers.length.toLocaleString() }}</strong> developers
```

Comment (ljcordero) [reply src/views/PopularDevelopers.vue:6]: I suppose that when virtual scroll get implemented `filteredDevelopers` will have all the developers that the first time, however I used `filteredDevelopers` cause when you use the filter I suppose would be better to count just the amount that really can be shown.

Comment (DannyFeliz) [reply src/views/PopularDevelopers.vue:6]: The virtual scroll is a mimic of showing all the developers, from the user perspective we're showing everything is just that they haven't scroll down enough to see all the developers. You can see [here](https://developersdo.github.io/opensource/developers/popular) how this is implemented.

Comment (ljcordero) [reply src/views/PopularDevelopers.vue:6]: Yes, but I for example someone filter to just those developers which name start with 'ger', we would just show those one, that was the reason why I used `filteredDevelopers` just to count those one that can be shown.

**Comment by rmariuzzo** [comment]
> I was talking with @rmariuzzo and he created a new [repository](https://github.com/developersdo/opensource-data) for the data and recently he added the repositories data, so, let's leave it like this for now until we fix the that.

The properties exported by https://github.com/developersdo/opensource-data for a user [are listed here](https://github.com/developersdo/opensource-data/blob/ec6c4b63d77e5f13c1cb323c2b58e29b4caf9148/src/scraper/users-scraper.ts#L49-L59).

**Comment by rmariuzzo** [comment]
> confirm me please if the 'forked' property on users.json is working

@ljcordero it is not for now.

**Pull request merged**